### ISSUE TITLE
Use Case 05: AS3 Install

### ISSUE THREAD

**Issue by codygreen** [f5alliances/isc-usecases#7]
Consider switching to the F5 App Services Package role in Ansible Galaxy: https://galaxy.ansible.com/f5devcentral/f5app_services_package

This is what we are directing customers towards now for Automation Toolchain install (DO, AS3, TS) when leveraging Ansible.

**Comment by danielcayer** [comment]
Also, when I ran through this lab just now, iRulesLX was not provisioned by this playbook. As Cody suggested above, these lab exercises should leverage DO to handles the provisioning tasks. The playbook in its current form runs and completes without error, but without iRulesLX being provisioned, the iApps->Packet Management LX menu is not available. Clicking to System >> Resource Provisioning and manually setting iRulesLX to "Nominal" makes the Packet Management LZX menu become available.

**Comment by payalsin** [comment]
@codygreen - will look at the official role
@danielcayer - ideally the module (bigip_iapplx_package )should enable iRulesLX and install the package. Which version of BIG-IP are you using, the one with the lab provisioner - 13.1?

**Comment by danielcayer** [comment]
@payal, yes: I strictly followed the instructions in the provisioner lab guide and version 13.1 was automatically deployed.

Sincères salutations,

Daniel Cayer
+1-613-868-1535<tel:+1-613-868-1535>  |  danielcayer@hotmail.com<mailto:danielcayer@hotmail.com>
